docker/docker-entrypoint.sh

```diff
@@ -122,27 +122,37 @@ install_languages() {
 	if [ ${#langs[@]} -eq 0 ]; then
 		return
 	fi
-	apt-get update
 
+	# Build list of packages to install
 	for lang in "${langs[@]}"; do
 		pkg="tesseract-ocr-$lang"
 
 		if dpkg --status "$pkg" &>/dev/null; then
 			echo "Package $pkg already installed!"
 			continue
+		else
+			to_install+=($pkg)
 		fi
+	done
 
-		if ! apt-cache show "$pkg" &>/dev/null; then
-			echo "Package $pkg not found! :("
-			continue
-		fi
+	# Use apt only when we install packages
+	if [ ${#to_install[@]} > 0 ]; then
```

Comment:
```suggestion
	if [ ${#to_install[@]} -gt 0 ]; then
```

Your version appears to work in bash, but I don't think it's posix. So it should be `[[ ]]` or `-gt`.

Reply:
good (and critical) catch! :see_no_evil: